src/torchcodec/decoders/_core/VideoDecoderOps.cpp

```diff
@@ -48,6 +48,7 @@ TORCH_LIBRARY(torchcodec_ns, m) {
       "get_frames_by_pts_in_range(Tensor(a!) decoder, *, int stream_index, float start_seconds, float stop_seconds) -> (Tensor, Tensor, Tensor)");
   m.def(
       "get_frames_by_pts(Tensor(a!) decoder, *, int stream_index, float[] timestamps) -> (Tensor, Tensor, Tensor)");
+  m.def("get_key_frame_indices(Tensor(a!) decoder, int stream_index) -> int[]");
```

Comment:
I think we can be conservative and expose it *privately* in the Python core API, for now? I.e. as `_get_key_frame_indices` instead of `get_key_frame_indices`